app/vfe-vdpa/vdpa_rpc.c

```diff
@@ -76,7 +87,10 @@ static void vdpa_rpc_get_info(cJSON *obj, uint64_t val, const char **info,
 	for (i = 0; i < size; i++) {
 		if (!(val & (1ULL << i)))
 			continue;
-		JSON_NUM_STR_TO_OBJ(obj, i, "%5d", info[i]);
+		if (info[i])
```

Comment:
IF i > sizeof feature_names,
it will cross mem

Reply:
Done